src/Items/ItemEyeOfEnder.h

```diff
@@ -49,7 +49,7 @@ class cItemEyeOfEnderHandler:
 					}
 					// Tries to spawn portal
 					auto ChunkInterface = cChunkInterface(a_World->GetChunkMap());
-					cBlockEndPortalFrameHandler::FindAndSetPortal(a_ClickedBlockPos, FacingMeta & 3, ChunkInterface, *a_World);
```

Comment:
Can you delete the #include "EndPortalFrame.h"?

Reply:
yes